src/components/EditCoursePage/EditCourseForm.jsx

```diff
@@ -33,13 +33,35 @@ export class BaseEditCourseForm extends React.Component {
     super(props);
 
     this.state = {
-      open: true,
+      open: false,
     };
 
     this.openCollapsible = this.openCollapsible.bind(this);
     this.setCollapsible = this.setCollapsible.bind(this);
   }
 
+  componentDidUpdate() {
+    const {
+      change,
+      courseInfo: {
+        courseSaved,
+      },
+      currentValues,
+      initialValues: {
+        imageSrc: initialImageSrc,
+        course_runs: initialCourseRuns,
+      },
+      updateFrontendAfterSave,
+    } = this.props;
+
+    if (courseSaved) {
+      // We need to overwrite the current form values because they are changed in the
+      // backend so frontend does not have the updated values by default. This will allow
+      // the form to return to being pristine after saving.
+      updateFrontendAfterSave(change, currentValues, initialImageSrc, initialCourseRuns);
```

Comment:
Nit: I do not think we need this comment in the code twice.  Within the method is probably enough.